lnwire/custom_records.go

```diff
@@ -0,0 +1,169 @@
+package lnwire
```

Comment:
are we keeping both this and `records` package version? (`record.CustomSet`)

Reply:
Yes, because of an import cycle. I think we could possibly replace `record.CustomSet` in the future.